### PR TITLE
Fix some more mypy errors

### PR DESCRIPTION
**Pull request by e-larsson**
Part of #51

### CODE REVIEW

File: scanomatic/models/compile_project_model.py [hunk 1]
```diff
@@ -85,11 +84,9 @@ class CompileImageAnalysisModelFields(Enum):
 class CompileImageAnalysisModel(Model):
     def __init__(
         self,
-        image: Optional[CompileImageModel] = None,
-        fixture: Optional[FixtureModel] = None,
+        image: CompileImageModel,
```
Comment (local-minimum) [inline scanomatic/models/compile_project_model.py:87]:
```suggestion
        *,
        image: CompileImageModel,
```
Models should only accept keyword arguments. But when I re-worked this, I didn't think of that they could be keyword arguments without defaults so if this works then perhaps you could update the other models to with the asterisk.

**Review by local-minimum — CHANGES_REQUESTED** [review]

File: scanomatic/models/compile_project_model.py (hunk 1 above)
Comment (e-larsson) [reply scanomatic/models/compile_project_model.py:87]: Some of the models caused errors in the factory when forcing keyword arguments, so I did this change only to the models that have falsely Optional parameters

Comment (local-minimum) [reply scanomatic/models/compile_project_model.py:87]: and this one caused issues?

Comment (e-larsson) [reply scanomatic/models/compile_project_model.py:88]: Oops, no this one was ok, fixed

**Review by local-minimum — APPROVED** [review]

**Pull request merged**